RecoLocalCalo/HcalRecProducers/python/hbheplan1.py

```diff
@@ -0,0 +1,21 @@
+import FWCore.ParameterSet.Config as cms
+
+hbheplan1 = cms.EDProducer(
```

Comment:
sorry for possibly unclear message earlier: _cfi should stay in the file name.
Thank you.